2015-digitas/index.html

```diff
@@ -1223,6 +1223,247 @@ <h3 id="codeofconductLabel">Code of conduct</h3>
+                      </p>
+
+                    </div>
+
+<div class="col-sm-2">
+
+                <div class="row speaker">
+
+                  <div class="col-sm-12 col-xs-6 speaker-image">
+
+                    <div class="hexagon-1" id="arron_faber"><div class="hexagon-2"><div class="hexagon-3"></div></div></div>
+
+                  </div>
+
+                  <div class="col-sm-12 col-xs-6 speaker-info">
+
+                    <p class="speaker-name">Aaron Faber</p>
+
+                    <p class="speaker-occupation">Senior Interface Designer<br />DigitasLBi</p>
+
+                  </div>
+
+                </div>
+
+              </div>
+
+                </div>
+
+            </div>
+
+        </div>
+
+        <div class="topic topic-talk">
+
+            <div class="wrapper">
+
+                <div class="row">
+
+
+                    <div class="col-sm-10">
+                        
+                                        <h3 class="title">Ian Huntington</h3>
+
+                        <p class="abstract">
+                        </p>
+
+                        <p class="link">
+                        </p>
+
+                        <p class="link">
+                        </p>
+
+                        <p class="bio">
+                          <b>About Ian</b> Ian is a front-end developer at DigitasLBi and has spent the past year building a RESTful API in Node and NodeWebkit apps. Ian was also a member of a team from DLBi that built a speaking clock for the Hack the Barbican exhibition. Away from writing code Ian plays the piano and attempts painting.
+                        </p>
+                    </div>
+
+                  <div class="col-sm-2">
+
+
+                    <div class="row speaker">
+
+                      <div class="col-sm-12 col-xs-6 speaker-image">
+
+                        <div class="hexagon-1" id="ian_huntington"><div class="hexagon-2"><div class="hexagon-3"></div></div></div>
+
+                      </div>
+
+                      <div class="col-sm-12 col-xs-6 speaker-info">
+
+                        <p class="speaker-name">Ian Huntington</p>
+
+                        <p class="speaker-occupation">Senior Interface Designer<br />DigitasLBi</p>
+
+                      </div>
+
+                    </div>
+
+                </div>
+
+            </div>
+
+        </div>
+            
+        </div>
+ 
+        <div class="topic topic-talk">
+
+            <div class="wrapper">
+
+                <div class="row">
+
+                    <div class="col-sm-10">
+
+                <h3 class="title">Richard Rodger</h3>
+                <!--<h3 class="title">Richard Rodger</h3>-->
```

Comment:
Probably shouldn't check in comments - especially along with the uncommented line right above it :)
Not a big deal here, @MikeZTaylor, but a good practice to watch out for.